apps/judicial-system/web/src/routes/Shared/AddFiles/AddFiles.tsx

```diff
@@ -142,19 +230,30 @@ const AddFiles: FC = () => {
           }
           nextButtonColorScheme={someFilesError ? 'destructive' : 'default'}
           nextIsDisabled={
-            uploadFiles.length === 0 || !allFilesDoneOrError || editCount > 0
+            uploadFiles.length === 0 ||
+            !allFilesDoneOrError ||
+            editCount > 0 ||
+            !hasValidFileRepresentativeSelection()
           }
-          onNextButtonClick={handleNextButtonClick}
+          onNextButtonClick={() => setVisibleModal('confirmation')}
```

Comment:
Moving the upload to the confirmation modal causes problems with failed uploads. Previously the user was given the option to retry if something failed. Now, we only get a toaster and navigate from this page without giving the user the option to retry.

Please reintroduce the previous behaviour.

Reply:
Nice.